docs/src/main/sphinx/release/release-418.md

```diff
@@ -0,0 +1,69 @@
+# Release 418 (17 May 2023)
+
+## General
+
+* Add support for `EXECUTE IMMEDIATE`. ({issue}`17341`)
+* Fix failure when invoking `current_timestamp`. ({issue}`17455`)
+
+## BigQuery connector
+
+* Add support for adding labels to BigQuery jobs started by Trino as part of
+  query processing. The name and value of the label can be configured via the
+  `bigquery.job.label-name` and `bigquery.job.label-format` catalog
+  configuration properties, respectively. ({issue}`16187`)
+
+## Delta Lake connector
+
+* Add support for `INSERT`, `UPDATE`, `DELETE`, and `MERGE` statements for
+  tables with an `id` column mapping. ({issue}`16600`)
+* Add `table_changes` table function. ({issue}`16205`)
```

Comment:
Add the table_changes table function .. also do we need to explain a bit what its for or link to docs for it ?

Reply:
Also not documented yet, so nothing to link to.